config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -112,8 +112,7 @@ presubmits:
         - --provider=gce
         # Skipping snat tests probably GCE related? https://github.com/kubernetes/test-infra/issues/20321
         # Skipping Cloud Provider specific tests: LoadBalancer, ESIPP (Source IP Preservation)
-        - >
-          --test_args=--ginkgo.skip=GCE|Disruptive|Serial|SNAT --ginkgo.label-filter=(sig-network || Conformance || Feature: containsAny {NetworkPolicy, NetworkPolicyEndPort} ) && !Feature: containsAny {Alpha, Beta, Example, Federation, IPv6DualStack, LoadBalancer, Networking-IPv6, PerformanceDNS, SCTPConnectivity, KubeProxyDaemonSetMigration}
+        - '--test_args=--ginkgo.skip=Disruptive|Serial|Alpha|Beta --ginkgo.label-filter="(sig-network || Conformance) && !Feature: containsAny {Alpha, Beta, Example, Federation, IPv6DualStack, LoadBalancer, Networking-IPv6, PerformanceDNS, SCTPConnectivity, KubeProxyDaemonSetMigration}"'
```

Comment:
this is how most of the jobs are running https://github.com/search?q=repo%3Akubernetes%2Ftest-infra+label-filter&type=code